src/components/Generator/GeneredTaskfile/SaveFile/SaveFile.tsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+
+import { ReactElement, useState } from 'react';
+
+import styles from './save-file.module.scss';
+
+type CopyProps = {
+	content: string;
+};
+
+const CopyToClipboard = ({ content }: CopyProps): ReactElement => {
```

Comment:
```suggestion
const SaveFile = ({ content }: CopyProps): ReactElement => {
```

Reply:
Fixed, thank  you for this valuable comment :+1: